SEARCHSPEC.md

```diff
@@ -0,0 +1,1476 @@
+   <td>Retrieve a paginated list of tables available from this Search API instance
+   </td>
+  </tr>
+  <tr>
+   <td>GET /table/{id}/info[^1]
+   </td>
+   <td>Retrieve the data model associated with the given table
+   </td>
+  </tr>
+  <tr>
+   <td>GET /table/{id}/data
+   </td>
+   <td>Retrieve the data model and data rows (paginated) from the given table
+   </td>
+  </tr>
+</table>
+
+
+
+#### Discovery and Browsing Examples
+
+
+```
+GET /tables
+```
+```
+{
+  "tables": [
+    {
+      "name": "search_cloud.brca_exchange.v32",
+      "data_model": {
+        "$ref": "https://ga4gh-search-adapter-presto-public.staging.dnastack.com/table/search_cloud.brca_exchange.v32/info"
+      }
+    },
+    {
+      "name": "search_postgres_pgpc.ontology.axiom",
+      "data_model": {
+        "$ref":     "https://ga4gh-search-adapter-presto-public.staging.dnastack.com/table/search_postgres_pgpc.ontology.axiom/info"
+        }
+    },
+    ...
+    ...
+  ],
+  "pagination": {
+    "next_page_url": "https://ga4gh-search-adapter-presto-public.staging.dnastack.com/tables/catalog/search_drs"
+  }
+}
+```
+
+```
+GET `/table/search_postgres_pgpc.ontology.axiom/info`
+```
+```
+{
+  "name": "search_postgres_pgpc.ontology.axiom",
+  "data_model": {
+    "$id": "https://ga4gh-search-adapter-presto-public.staging.dnastack.com/table/search_postgres_pgpc.ontology.axiom/info",
+    "description": "Automatically generated schema",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+      "ontology": {
+        "format": "varchar",
+        "type": "string",
+      },
+      "ontology_version": {
+        "format": "varchar",
+        "type": "string",
+      },
+      "from_term": {
+        "format": "varchar",
+        "type": "string",
+      },
+      "relation": {
+        "format": "varchar",
+        "type": "string",
+      },
+      "to_term": {
+        "format": "varchar",
+        "type": "string",
+      }
+    }
+  }
+}
+```
+
+
+
+```
+GET /table/`search_postgres_pgpc`.ontology.axiom/data
+```
+```
+{
+  "data_model": { <described in the /info example above> },
+  "data": [
+    {
+      "ontology": "http://purl.obolibrary.org/obo/hp.owl",
+      "ontology_version": "http://purl.obolibrary.org/obo/hp/releases/2019-04-15",
+      "from_term": "HP_0100572",
+      "relation": "SubClassOf",
+      "to_term": "HP_0100571"
+    },
+    {
+      "ontology": "http://purl.obolibrary.org/obo/hp.owl",
+      "ontology_version": "http://purl.obolibrary.org/obo/hp/releases/2019-04-15",
+      "from_term": "UBERON_0005047",
+      "relation": "SubClassOf",
+      "to_term": "UBERON_0001824"
+    },
+    ...
+    ...
+  ],
+  "pagination": {
+    "next_page_url": "https://ga4gh-search-adapter-presto-public.staging.dnastack.com/search/v1/statement/executing/20200901_013328_00039_szpff/y4134d1e51a8262d0f8fed899b2eed9fd02e200e9/1"
+}
+```
+
+### Query
+
+The Query part of the Search API consists of the following REST operation:
+
+```
```

Comment:
This table is rendering as HTML source code. I think it was meant to render as a table.